core/loadpoint/config.go

```diff
@@ -17,17 +17,17 @@ type StaticConfig struct {
 
 type DynamicConfig struct {
 	// dynamic config
-	Title          string    `json:"title"`
-	DefaultMode    string    `json:"defaultMode"`
-	Priority       int       `json:"priority"`
-	Phases         int       `json:"phases"`
-	MinCurrent     float64   `json:"minCurrent"`
-	MaxCurrent     float64   `json:"maxCurrent"`
-	SmartCostLimit *float64  `json:"smartCostLimit"`
-	PlanEnergy     float64   `json:"planEnergy"`
-	PlanTime       time.Time `json:"planTime"`
-	LimitEnergy    float64   `json:"limitEnergy"`
-	LimitSoc       int       `json:"limitSoc"`
+	Title            string    `json:"title"`
+	DefaultMode      string    `json:"defaultMode"`
+	Priority         int       `json:"priority"`
+	PhasesConfigured int       `json:"phasesConfigured"`
```

Comment:
brauchen wir Phases nicht weiter damit die zwischenzeitlichen Fehler restored werden können?

Reply:
Du meinst die während des Nightlies erzeugt wurden? Released is ja noch nichts. Würd ich nicht machen. Dann muss man halt den DB-Eintrag löschen. Mit der Änderung oben (Reihenfolge geändert) sollte das sogar durch die UI im Fehlerfall möglich sein 😁

Reply:
Uhhhh, long shot. Super!